packages/cli/test/integration-1.test.ts

```diff
@@ -750,18 +753,19 @@ test.skip('deploy `api-env` fixture and test `vercel env` command', async () =>
     expect(apiRes.status).toBe(200);
 
     const apiJson = await apiRes.json();
-    expect(apiJson['MY_NEW_ENV_VAR']).toBe('my plaintext value');
-    expect(apiJson['MY_STDIN_VAR']).toBe('{"expect":"quotes"}');
+    expect(apiJson[promptEnvVar]).toBe('my plaintext value');
+    expect(apiJson[stdinEnvVar]).toBe('{"expect":"quotes"}');
 
     const homeUrl = localhost[0];
     const homeRes = await fetch(homeUrl);
     const homeJson = await homeRes.json();
-    expect(homeJson['MY_NEW_ENV_VAR']).toBe('my plaintext value');
-    expect(homeJson['MY_STDIN_VAR']).toBe('{"expect":"quotes"}');
+    expect(homeJson[promptEnvVar]).toBe('my plaintext value');
+    expect(homeJson[stdinEnvVar]).toBe('{"expect":"quotes"}');
 
     // system env vars are hidden in dev
     expect(apiJson['VERCEL']).toBeUndefined();
-    expect(homeJson['VERCEL']).toBeUndefined();
+    // though the dev server now has this
+    expect(homeJson['VERCEL']).toBe('1');
```

Comment:
Not completely sure what the different between `apiJson` and `homeJson` is here, but https://github.com/vercel/vercel/pull/12358 makes `VERCEL` available in `homeJson`